vicinity/integrations/huggingface.py

```diff
@@ -131,8 +151,7 @@ def load_from_hub(cls, repo_id: str, token: str | None = None, **kwargs: Any) ->
             config = json.load(f)
             model_name_or_path = config.pop("model_name_or_path")
 
-        print(_MODEL_NAME_OR_PATH_PRINT_STATEMENT.format(model_name_or_path=model_name_or_path))
+        print(_MODEL_NAME_OR_PATH_PRINT_STATEMENT.format(model_name_or_path=model_name_or_path))  # noqa: T201
```

Comment:
I'm a bit confused as to why this is here. I don't remember, but shouldn't this be a logging message?